src/common/Notifications.ts

```diff
@@ -34,53 +50,59 @@ export const registerDeviceToken = ({
   httpClient.post('/v1/device-endpoints', params, options);
 };
 
-export const getInitialNotification = () => {
-  return Notifications.getInitialNotification();
+export const getInitialNotification = async () => {
+  const { rnnotifications } = await safelyImportReactNativeNotifications();
+  return rnnotifications.Notifcations.getInitialNotification();
 };
 
-export const isRegisteredForNotifications = () => {
-  return Notifications.isRegisteredForRemoteNotifications();
+export const isRegisteredForNotifications = async () => {
+  const { rnnotifications } = await safelyImportReactNativeNotifications();
+  return rnnotifications.Notifications.isRegisteredForRemoteNotifications();
 };
 
-export const requestNotificationsPermissions = (
+export const requestNotificationsPermissions = async (
   callback: ({
     deviceToken,
     denied,
     error,
   }: {
     deviceToken?: string;
     denied?: boolean;
-    error?: RegistrationError;
+    error?: any;
```

Comment:
i would have preferred not to use `any` here and below but there is no way to dynamically import types that i know of. Since we are relying on dynamic imports, my hand was forced. Maybe you guys know of a mechanism around this?